src/components/ebay-carousel/utils/on-scroll-end/index.js

```diff
@@ -40,17 +41,19 @@ module.exports = function onScrollEnd(el, fn) {
     // Finally after the touch end we poll the scroll state every animation frame
     // to determine when inertial scrolling has stopped.
     function checkScrollEnded(lastOffset) {
-        frame = requestAnimationFrame(() => {
```

Comment:
An additional 64ms timeout seems to resolve the issue in even the slowest devices. (Tested on the Moto G5 Plus).